src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2388,6 +2388,236 @@ curl -X POST \
+
+Users with `tags--all:rw` scope can create/delete/update tags in all license's groups. Users with `tags--groups:rw` scope are restricted to groups they are assigned to. Similarly `tags--all:ro` and `tags--groups:ro` scopes affect listing tags.
+
+Tag can be used to mark a chat's thread only if the thread and the tag are assigned to at least one common group.
+
+</Text>
+
+</Section>
+
+## Methods
+
+| HTTP method | The Tags API endpoint                                            |
+| ----------- | ---------------------------------------------------------------- |
+| `POST`      | `https://api.livechatinc.com/v3.5/configuration/action/<action>` |
+
+| Required header |       Value        |      |
+| --------------- | :----------------: | ---: |
+| `Content-Type`  | `application/json` |      |
+
+<Section>
+
+<Text>
+
+### Create Tag
+
+Creates a new tag.
+
+#### Specifics
+
+|                 |                                                                    |
+| --------------- | ------------------------------------------------------------------ |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/create_tag` |
+| Required scopes | `tags--all:rw, tags--groups:rw`                                    |
+
+#### Request
+
+| Parameter   | Required | Data type | Notes                                                     |
+| ----------- | -------- | --------- | --------------------------------------------------------- |
+| `name`      | yes      | `string`  | Matching the name of an existing tag is case-insensitive. |
+| `group_ids` | yes      | `[]int`   | Can be empty.                                             |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/create_tag \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "name": "docs_feedback",
+    "group_ids": [
+      0,
+      42
+    ]
+  }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Delete Tag
+
+Deletes an existing tag.
+
+#### Specifics
+
+|                 |                                                                    |
+| --------------- | ------------------------------------------------------------------ |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/delete_tag` |
+| Required scopes | `tags--all:rw, tags--groups:rw`                                    |
+
+#### Request
+
+| Parameter | Required | Data type | Notes                                                   |
+| --------- | -------- | --------- | ------------------------------------------------------- |
+| `name`    | yes      | `string`  | Matching the name of an existing tag is case-sensitive. |
```

Comment:
```suggestion
| `name`    | yes      | `string`  | Matching the name of an existing tag is case sensitive. |
```

Reply:
This is correct, but it's more popular in BrE and we lean toward AmE

Reply:
"whatever-keyboard-accepts" English is the best English :trollface: